src/completion/completionProvider.ts

```diff
@@ -44,8 +45,9 @@ class CompletionProvider implements vscode.CompletionItemProvider {
                 const artifactIdHint: string = artifactIdNode && artifactIdNode.text ? artifactIdNode.text : "";
 
                 const centralItems: vscode.CompletionItem[] = await centralProvider.getGroupIdCandidates(groupIdHint, artifactIdHint);
+                const indexItems: vscode.CompletionItem[] = await indexProvider.getGroupIdCandidates(groupIdHint, artifactIdHint);
                 const localItems: vscode.CompletionItem[] = await localProvider.getGroupIdCandidates(groupIdHint, artifactIdHint);
-                const mergedItems: vscode.CompletionItem[] = this.deDuplicate(centralItems, localItems);
+                const mergedItems: vscode.CompletionItem[] = this.deDuplicate(this.deDuplicate(centralItems, indexItems), localItems);

```

Comment:
consider change the signature of `deDuplicate(a, b)` to `deDuplicate(...arr)`?

Reply:
I finds `_.unionBy` api supports union three arrays at the same time. Is it necessary to change it?

Reply:
never mind, I'm also ok with current code.